apps/storybook/stories/components/editable/Editable.mdx

```diff
@@ -1,13 +1,23 @@
-import { Meta } from "@storybook/blocks";
+import { Meta, Canvas, Story, Controls } from "@storybook/blocks";
+import * as EditableStories from "./Editable.stories";
 
 <Meta title="Komponenter/Editable" />
```

Comment:
Bytt ut denne linja med 
Meta of={EditableStories} 
bortsett fra det ser det veldig bra ut 😄

Reply:
Hmm der har du et godt poeng! Det fungerer med det som er der, noe som er litt rart